src/main/java/com/nirima/jenkins/plugins/docker/DockerTemplateBase.java

```diff
@@ -168,6 +169,7 @@ public DockerTemplateBase(String image) {
      * @param extraGroupsString  See {@link #setExtraGroupsString(String)}
      * @param memoryLimit        See {@link #setMemoryLimit(Integer)}
      * @param memorySwap         See {@link #setMemorySwap(Integer)}
+     * @param cpus               See {@link #setCpus(String)}
```

Comment:
Unwanted line, please remove.
You've not added a parameter "for real" (which is correct - this is a deprecated method and shouldn't be changed) so you shouldn't document the addition of the new parameter.